telegapi/utils.py

```diff
@@ -33,10 +33,14 @@ def msg_get_summary(msg, truncate=0):
     summary = ""
 
     if 'text' in msg:
-        summary += "[Text: " +\
-                   ((msg["text"][:truncate] + '...')
-                    if (len(msg["text"]) > truncate and truncate is not 0)
-                    else msg["text"]) + "] "
+
+        message = msg["text"]
+        if (0 < truncate < len(message)):
+            summary = message[:truncate] + '...'
+        else:
+            summary = message + ']'
+
+        summary = "[Text: " + str(summary.encode('unicode_escape'))
```

Comment:
Couldn't we check if an emoji is included and print "emoji" or something instead of re-encoding?